RunLog/RunLog/Sources/Presentation/Log/View/DetailLogView.swift

```diff
@@ -0,0 +1,303 @@
+    )
+    
+    private let timeValueLabel = RLLabel(
+        text: "24시간 23분",
+        textColor: .LightOrange,
+        icon: nil,
+        align: .left,
+        font: .RLHeading1
+    )
+    
+    private lazy var timeStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [timeTitleLabel, timeValueLabel])
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.spacing = 4
+        return stack
+    }()
+    
+    private let distanceTitleLabel = RLLabel(
+        text: "운동거리",
+        textColor: .Gray000,
+        icon: nil,
+        align: .left,
+        font: .RLBody1
+    )
+    
+    private let distanceValueLabel = RLLabel(
+        text: "9999.99km",
+        textColor: .LightPink,
+        icon: nil,
+        align: .left,
+        font: .RLHeading1
+    )
+    
+    private lazy var distanceStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [distanceTitleLabel, distanceValueLabel])
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.spacing = 4
+        return stack
+    }()
+    
+    private let stepsTitleLabel = RLLabel(
+        text: "걸음수",
+        textColor: .Gray100,
+        icon: nil,
+        align: .left,
+        font: .RLBody1
+    )
+    
+    private let stepsValueLabel = RLLabel(
+        text: "2,237,345",
+        textColor: .LightBlue,
+        icon: nil,
+        align: .left,
+        font: .RLHeading1
+    )
+    
+    private lazy var stepsStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [stepsTitleLabel, stepsValueLabel])
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.spacing = 4
+        return stack
+    }()
+    
+    private lazy var statsStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [timeStack, distanceStack, stepsStack])
+        stack.axis = .horizontal
+        stack.alignment = .center
+        stack.spacing = 16
+        stack.distribution = .fillEqually
+        return stack
+    }()
+    
+    
+    // MARK: - Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+        setupLayout()
+        print("LogView initialized")
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Setup UI
+    private func setupUI() {
+        // UI 요소 추가
+        backgroundColor = .Gray900
+        // 1) 스크롤뷰 & contentView
+        addSubview(scrollView)
+        scrollView.addSubview(contentView)
+        
+        // 2) 지도 + 무빙트랙 버튼
+        contentView.addSubview(mapView)
+        mapView.addSubview(movingTrackButton)
+        
+        // 3) 타이틀, 날씨 스택
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(weatherStack)
+        
+        // 4) 구분선
+        contentView.addSubview(separatorView)
+        
+        // 5) 통계 스택
+        contentView.addSubview(statsStack)
```

Comment:
현재 contentView에서 따로 따로 넣고 있는데
@doyeonk429 가 만드신 addSubviews를 이용해보시는 건 어떨까요?
```suggestion
        contentView.addSubviews(mapView, titleLabel, weatherStack, ... )
```